swag-chart/templates/accounts_deploy.yaml

```diff
@@ -1,31 +1,36 @@
-{{- with .Values.accounts }}
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: {{ .deploymentName }}
-spec:
-  replicas: {{ .replicas }}
-  selector:
-    matchLabels:
-      app: {{ .app }}
-  template:
-    metadata:
-      labels:
-        app: {{ .app }}
-    spec:
-      containers:
-      - name: {{ .deploymentName }}
-        image: {{ .image }}:{{ .tag }}
-        envFrom:
-        - configMapRef:
-            name: env-vars
-        resources:
-          requests:
-            memory: {{ quote .memoryRequest }}
-            cpu: {{ quote .cpuRequest }}
-          limits:
-            memory: {{ quote .memoryLimit }}
-            cpu: {{ quote .cpuLimit }}
-        ports:
-        - containerPort: {{ .containerPort }}
-{{ end }}
+{{- with .Values.accounts }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .deploymentName }}
+  labels:
+    app: {{ .app }}
+    role: {{ .role }}
+spec:
+  replicas: {{ .replicas }}
+  selector:
+    matchLabels:
+      app: {{ .app }}
+      role: {{ .role }}
+  template:
+    metadata:
+      labels:
+        app: {{ .app }}
+        role: {{ .role }}
+    spec:
+      containers:
+      - name: {{ .deploymentName }}
+        image: {{ .image }}:{{ .tag }}
+        envFrom:
+        - configMapRef:
+            name: env-vars
+        resources:
+          requests:
+            memory: {{ quote .memoryRequest }}
+            cpu: {{ quote .cpuRequest }}
+          limits:
+            memory: {{ quote .memoryLimit }}
+            cpu: {{ quote .cpuLimit }}
+        ports:
+        - containerPort: {{ .containerPort }}
+{{ end }}
```

Comment:
i don't understand why github is failing to show these diffs (the ones where i look at only the changes of the last 2 commits, since my last review) intelligently when other tools are doing fine. ideally you would figure out why and work around the problem, because it makes code review difficult.